docs/src/user_guide/install.md

```diff
@@ -0,0 +1,122 @@
+# How to install
+
+Wflow can be used in two different ways, depending on the required use of the code:
+
+- If you want to stay up-to-date with the latest version, explore and modify the model code,
+  and write your own Julia scripts around the wflow package, we recommend installing wflow
+  as a [Julia](https://julialang.org/) package.
+- If you don't need extra features, but just want to run simulations, a complied executable
+  version is available. This consists of a single executable, `wflow_cli`, allowing you to
+  run the model via the command line.
+
+Below, we describe how to install both versions of wflow. 
+
+## Installing as Julia package
```

Comment:
We also use to have info on how to install Julia itself and some intro how it works in the old quick guide. As Julia is still new to a lot of people, I think maybe useful to keep that info.

Reply:
I felt like the section on installing an using Julia was a bit redundant to keep in the documentation for a package of Julia, so I now added a link to a quick start guide from julialang.org

Reply:
Agreed but from experience with hydromt-wflow (python) I get a lot of questions on installation if we just say the classic "conda install -c conda-forge hydromt-wflow"...
So we in the end added some lines also on first download miniconda and create an empty environment.
Saves you some trouble with brand new julia users :)

Your additions are good but I would add the previous lines with the install julia link as well. The rest can go I agree.
"First download and install the [current stable release of Julia](https://julialang.org/downloads/#current_stable_release). Please see [platform specific instructions](https://julialang.org/downloads/platform/) for further installation instructions and if you have trouble installing Julia."

Reply:
That is a valid point. The links to the Julia install instructions are now added to the docs.